Scrypted/Scrypted.php

```diff
@@ -0,0 +1,35 @@
+<?php namespace App\SupportedApps\Scrypted;
+
+class Scrypted extends \App\SupportedApps implements \App\EnhancedApps {
+
+    public $config;
+
+    //protected $login_first = true; // Uncomment if api requests need to be authed first
+    //protected $method = 'POST';  // Uncomment if requests to the API should be set by POST
+
+    function __construct() {
+        //$this->jar = new \GuzzleHttp\Cookie\CookieJar; // Uncomment if cookies need to be set
+    }
+
+    public function test()
+    {
+        $test = parent::appTest($this->url('status'));
+        echo $test->status;
+    }
+
+    public function livestats()
+    {
+        $status = 'inactive';
+        $res = parent::execute($this->url('status'));
+        $details = json_decode($res->getBody());
+
+        $data = [];
+        return parent::getLiveStats($status, $data);
```

Comment:
This is the template for enhanced app, however it is missing the implementation part here.